_pages/general/sport.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Sport
+---
+
+# Sport
+Australia has a pluralistic sports culture. There's no _national sport_ and some sports are enjoyed more than others depending on who you speak to.
+
+Australians will enjoy Tennis, Cricket, Swimming, Water Polo, Surfing, Hockey, Rugby, Netball, Basketball, and Australian Rules Football (this is a non-exhaustive list).
+
+## Football
```

Comment:
Good work on this Raph. Would it be worth adding swimming and gym to the list? e.g. popular Gyms or aquatic centres and maybe some recommendations

Reply:
sure I'm happy to do this. Can I do it in another PR though?

Reply:
I'm assuming you mean like popular franchises like Fitness First etc?

Reply:
> sure I'm happy to do this. Can I do it in another PR though?

Yeah syre

Reply:
> I'm assuming you mean like popular franchises like Fitness First etc?

Yeah (Y)